src/styles/theme/oc-progress.scss

```diff
@@ -0,0 +1,75 @@
+// Variables
+// ========================================================================
+
+$progress-height: 15px !default;
+$progress-height-small: 5px !default;
+
+// Component
+// ========================================================================
+
+.oc-progress {
+    // Add the correct vertical alignment in Chrome, Firefox, and Opera.
+    vertical-align: baseline;
+    // Remove default style
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    display: block;
+    width: 100%;
+    // Remove default style
+    border: 0;
+    // Set background color for progress container in Firefox, IE11 and Edge
+    background-color: $global-inverse-color;
+    height: $progress-height;
+
+    &:focus {
+      outline: none;
+    }
+
+    // Remove animated circles for indeterminate state in IE11 and Edge
+    &:indeterminate {
+      color: transparent;
+    }
+
+    &::-webkit-progress-bar {
+      background-color: $global-inverse-color;
+    }
+
+    // Remove progress bar for indeterminate state in Firefox
+    &:indeterminate::-moz-progress-bar {
+      width: 0;
+    }
+
+    &::-webkit-progress-value {
+      background-color: $action-primary;
+      transition: width $transition-duration-short ease;
+    }
+
+    &::-moz-progress-bar {
+      background-color: $action-primary;
+    }
+
+    &::-ms-fill {
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=5920039)